src/booking/dto/create-booking.dto.ts

```diff
@@ -1 +1,30 @@
-export class CreateBookingDto {}
+import { IsDate, IsInt, IsString } from 'class-validator';
+
+export class CreateBookingDto {
+  @IsDate()
+  startDate: Date;
+
+  @IsDate()
```

Comment:
add validations correctly startDate should be greater than currentDate , endDate >= startDate , numNights > =0 , price?=0